bin/uberjar

```diff
@@ -2,13 +2,14 @@
 
 set -eux
 
-export build_data="$(mktemp -d)"
+build_data="$(mktemp -d)"
+export build_data
```

Comment:
I'm not sure if this `export` is needed in this script?

Reply:
I think you're right, this is a vestige of an earlier approach.